ql/src/java/org/apache/hadoop/hive/ql/DriverFactory.java

```diff
@@ -73,7 +73,7 @@ private static IReExecutionPlugin buildReExecPlugin(String name) throws RuntimeE
     if("reexecute_lost_am".equals(name)) {
       return new ReExecuteLostAMQueryPlugin();
     }
-    if (name.equals("dagsubmit")) {
+    if ("dagsubmit".equals(name)) {
```

Comment:
This is a good change, but please revert for this PR to keep it focused on the Driver classes.

Reply:
OK, removed.